src/components/modals/GradientFadeout.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+import { cacheStyles } from 'react-native-patina'
+
+import { Theme, useTheme } from '../services/ThemeContext'
+
+const MARKS: number[] = [0, 0.2, 0.75, 1]
+const START = { x: 0, y: 0 }
+const END = { x: 0, y: 1 }
+
+/*
+ * Used for adding a gradient fadeout to the bottom of a list modal
+ */
+export function GradientFadeOut() {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  const color = theme.modal
+  const colors: string[] = MARKS.map(mark => color + `0${Math.floor(255 * mark).toString(16)}`.slice(-2))
```

Comment:
I realized I missed this on the login-ui PR, but colors should be memoized otherwise you're constantly re-rendering.